Board_Game_App.py

```diff
@@ -74,13 +73,10 @@ def get_player_rating(game: str, player_name: str):
     if player_name not in app.games_by_player.keys():
         raise HTTPException(status_code=404, detail=f"Player {player_name} not found.")
 
-    game_list = [player_game["Game"] for player_game in app.games_by_player[player_name]]
-    if game not in game_list:
+    if game not in app.games_by_player[player_name].keys():
         raise HTTPException(status_code=404, detail=f"Game {game} not rated by this player.")
 
-    player_rating = next(player_game for player_game in app.games_by_player[player_name] if player_game["Game"] == game)
-
-    return player_rating["Rating"]
+    return app.games_by_player[player_name][game]
```

Comment:
Awesome - you took it down from two iterations to 0 🎖️

This is one reason why considering carefully the format of data you attach to the app, can have a big impact - imagine this is a worldwide service that millions of people use - the impact can add up!